File: src/com/gk/mortgage/calculator/domain/MortgageCalculatorResponse.java
package com.gk.mortgage.calculator.domain;

public class MortgageCalculatorResponse {

    private double monthlyPayment;

    public MortgageCalculatorResponse(double monthlyPayment) {
        this.monthlyPayment = monthlyPayment;
    }

    public double getMonthlyPayment() {
        return monthlyPayment;
    }

    public void setMonthlyPayment(double monthlyPayment) {
        this.monthlyPayment = monthlyPayment;
    }
}

File: webapp/domain/MortgageCalculatorResponse.js


function MortgageCalculatorResponse(monthlyPayment) {
    'use strict';

    // holder for privileged methods.
    var _self = this;

    _self.getMonthlyPayment = () => {
        return monthlyPayment;
    }

    _self.setMonthlyPayment = (monthlyPayment) => {
        this.monthlyPayment = monthlyPayment;
    }
}
